lms/djangoapps/instructor_task/api.py

```diff
@@ -63,7 +69,23 @@ def get_running_instructor_tasks(course_id):
     # exclude states that are "ready" (i.e. not "running", e.g. failure, success, revoked):
     for state in READY_STATES:
         instructor_tasks = instructor_tasks.exclude(task_state=state)
-    return instructor_tasks.order_by('-id')
+
+    instructor_tasks = instructor_tasks.order_by('-id')
+
+    # list down remote grade book tasks
+    now = datetime.datetime.now()
+    rgb_tasks = InstructorTask.objects.filter(
+        course_id=course_id,
+        task_state__icontains="success",
+        task_type=TASK_TYPE_EXPORT_GRADES_TO_RGB,
+        updated__lte=now,
+        updated__gte=now - datetime.timedelta(days=5)
+    ).order_by('-id')
+
+    if user is not None:
+        rgb_tasks = rgb_tasks.filter(requester=user)
+
+    return (instructor_tasks | rgb_tasks).distinct()
```

Comment:
Instead of modifying the existing `get_running_instructor_tasks`, I think we should do the following:

1. Create a new function here that gets the running tasks from the original `get_running_instructor_tasks`, then adds on the successful grade export tasks like you're doing here. That can be called `get_running_instructor_rgb_tasks` or something. That function would require a `user` argument.
1. When you create the url for `list_instructor_tasks_url` in `instructor.views.instructor_dashboard._section_remote_gradebook`, define it like this to add a querystring parameter:
    ```python
    'list_instructor_tasks_url': '{}?include_remote_gradebook=true'.format(
            reverse(
                'list_instructor_tasks', kwargs={'course_id': unicode(course.id)}
            )
        )
    ```
1. In `instructor.views.api.list_instructor_tasks`, check for that querystring variable. If it exists, then set `tasks = lms.djangoapps.instructor_task.api.get_running_instructor_rgb_tasks(...)` (or whatever you call that new function). It would look something like this:
    ```python
    def list_instructor_tasks(request, course_id):
        course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
        # ...
        include_remote_gradebook = request.REQUEST.get('include_remote_gradebook') is not None
        if problem_location_str:
            # ...
        elif include_remote_gradebook:
            tasks = lms.djangoapps.instructor_task.api.get_running_instructor_rgb_tasks(...)
        else:
            tasks = lms.djangoapps.instructor_task.api.get_running_instructor_tasks(...)
        # ...
    ```

That will avoid returning those 'success' tasks that other tabs don't care about